projectroles/forms.py

```diff
@@ -957,6 +957,30 @@ def clean(self):
         except ProjectInvite.DoesNotExist:
             pass
 
+        # Local users check
+        try:
+            user_email = self.cleaned_data.get('email')
+            domain = (
+                user_email[user_email.find('@') + 1 :].split('.')[0].lower()
+            )
+            domain_list = [
+                getattr(settings, 'ALT_LDAP_DOMAINS', ''),
```

Comment:
This should be `LDAP_ALT_DOMAINS`. Also note that, as I described in my comment before, the syntax for `LDAP_ALT_DOMAINS` is different. There we have the entire domain with the TLD (e.g. "example.com") while in the primary auth domains we have it without the TLD (e.g. "EXAMPLE").